joomla/mod_krizalys_breadcrumbs/language/hu-HU/hu-HU.mod_krizalys_breadcrumbs.sys.ini

```diff
@@ -0,0 +1,8 @@
+; Krizalys
+; Copyright (C) 2008 - 2019 Krizalys (http://www.krizalys.com/). All rights reserved.
+; GNU General Public License version 3 (http://www.gnu.org/licenses/gpl.html).
+; Note: All ini files need to be saved as UTF-8
+
+MOD_KRIZALYS_BREADCRUMBS="Krizalys Breadcrumbs"
+MOD_KRIZALYS_BREADCRUMBS_LAYOUT_DEFAULT="Default"
```

Comment:
Does this really translate into `Default` in Hungarian?

Reply:
No, but I couldn't find the context where it's used and without that it's not possible to accurately translate it. I can only push a generic translation of that word this way

Reply:
@belidzs Fair point.

This is the label of the default module layout. It shows when one tries to change the module layout, on its advanced tab:

![Advanced options](https://i.imgur.com/xIec8Gp.png)

I believe other built-in modules are using the same label, perhaps we can reuse what the same Hungarian translation from one of them.

Reply:
The proposed changes in 162bf60 will work in this context